src/test/java/addtoPlaylist.java

```diff
@@ -0,0 +1,50 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.interactions.Actions;
+
+import java.security.Key;
+import java.time.Duration;
+
+public class addtoPlaylist extends BaseTest {
+    @Test
+    public static void addSongtoPlaylist() {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        WebElement emailadress = driver.findElement(By.cssSelector("[type='email']"));
+        emailadress.click();
+        emailadress.sendKeys("demotesting@gmail.com");
+        WebElement passwordfield = driver.findElement(By.cssSelector("[type='password']"));
+        passwordfield.click();
+        passwordfield.sendKeys("te$t$tudent");
+        WebElement submitButton = driver.findElement(By.cssSelector("button[type='submit']"));
+        submitButton.click();
+        WebElement avatar = driver.findElement(By.cssSelector(".avatar"));
+        Assert.assertTrue(avatar.isDisplayed());
+        WebElement AllSongs = driver.findElement(By.xpath("//*[@class='songs']"));
+        AllSongs.click();
+        WebElement Favorite = driver.findElement(By.xpath("//tr[1]/td[6]/button/i"));
+        Favorite.click();
+        WebElement favoritetab = driver.findElement(By.xpath("//section[@id='playlists']/ul//a[@href='#!/favorites']"));
+        favoritetab.click();
+        WebElement song = driver.findElement(By.xpath("//section[@id='favoritesWrapper']//div[@class='item-container']/table[@class='items']//td[@class='title']"));
+        driver.quit();
+
+    }
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._